lib/src/widget/nav_bar/widget/nav_bar_item/icon_button.dart

```diff
@@ -13,7 +13,7 @@ class NavBarIconButton extends StatelessWidget {
       data.svgIconPath,
       width: linden.dimen.iconSize,
       height: linden.dimen.iconSize,
-      color: linden.colors.icon,
+      color: data.isDisabled ? linden.colors.iconDisabled : linden.colors.icon,
```

Comment:
❓ don't you want to disable `onPressed` (set to null) if `data.isDisabled`?

Reply:
We don't want to do that because for the active search button we still want to handle `onPressed` to show tooltip.